File: csharp/ButtplugCSharpFFI/ButtplugConnectorException.cs
using System;

namespace Buttplug
{
    public class ButtplugConnectorException : ButtplugException
    {
        /// <inheritdoc />
        public ButtplugConnectorException()
        {
        }

        /// <inheritdoc />
        public ButtplugConnectorException(string aMessage) : base(aMessage)
        {
        }

        /// <inheritdoc />
        public ButtplugConnectorException(string aMessage, Exception aInner) : base(aMessage, aInner)
        {
        }
    }
}

File: java/lib/src/main/java/io/buttplug/ButtplugConnectorException.java
package io.buttplug;

public class ButtplugConnectorException extends ButtplugException {
    public ButtplugConnectorException(String msg) {
        super(msg);
    }

    public ButtplugConnectorException(String msg, Exception inner) {
        super(msg, inner);
    }
}
